src/ensembl/src/content/app/entity-viewer/state/sidebar/sampleData.ts

```diff
@@ -0,0 +1,749 @@
+import { EntityViewerSidebarPayload } from './entityViewerSidebarState';
+import { DataSetType } from '../../types/dataSet';
+
+export const entityViewerWheatSampleResponse: EntityViewerSidebarPayload = {
+  gene: {
+    symbol: 'TraesCS3D02G273600.1 ',
+    name: 'Heat shock protein 101',
+    id: 'ENSG00000139618.15',
+    xrefs: [
+      {
+        source_name: 'KNETMINER_WHEAT',
+        links: [
+          {
+            name: 'KNETMINER_WHEAT',
+            value: 'TraesCS3D02G273600',
+            url: ''
+          }
+        ]
+      },
+      {
+        source_name: 'Wheat Expression',
+        links: [
+          {
+            name: 'Wheat Expression',
+            value: 'TraesCS3D02G273600',
+            url: ''
+          }
+        ]
+      }
+    ],
+    transcripts: [
+      {
+        id: 'TraesCS3D02G273600.1'
+      },
+      {
+        id: 'TraesCS3D02G273600.2'
+      }
+    ],
+    filters: {
+      transcript: {
+        sequence: [
+          {
+            value: 'Genomic sequence',
+            label: 'Genomic sequence'
+          },
+          {
+            value: 'cDNA',
+            label: 'cDNA'
+          },
+          {
+            value: 'CDS',
+            label: 'CDS'
+          },
+          {
+            value: 'Protein sequence',
+            label: 'Protein sequence'
+          }
+        ],
+        tark_url: 'http://betatark.ensembl.org/web/search/'
+      }
+    }
+  },
+  other_assemblies: [
+    {
+      species_name: 'Bread wheat ',
+      assembly_name: 'ASM34745v1',
+      stable_id: 'TRIUR3_14529'
+    },
+    {
+      species_name: 'Durum wheat',
+      assembly_name: 'Svevo.v1',
+      stable_id: 'TRITD4Av1G160860'
+    },
+    {
+      species_name: 'Emmer wheat',
+      assembly_name: 'WEWSeq_v.1.0',
+      stable_id: 'TRIDC4BG042680'
+    }
+  ],
+  other_data_sets: [
+    { type: DataSetType.CURRENT_ASSEMBLY, value: 'IWGSC' },
+    { type: DataSetType.GENE, value: 'TraesCS3D02G273600.1' },
+    { type: DataSetType.PROTEIN, value: 'TraesCS3D02G273600.1' }
+  ],
+  homeologues: [
+    {
+      type: 'Gene',
+      stable_id: 'TraesCS3A02G274400'
+    },
+    {
+      type: 'Gene',
+      stable_id: 'TraesCS3B02G308100'
+    }
+  ],
+  publications: [
+    {
+      title:
+        'Long noncoding RNA LINC00520 accelerates progression of papillary thyroid carcinoma by serving as a competing endogenous RNA of microRNA-577 to increase Sphk2 expression.',
+      description: 'Sun Y, Shi T, Ma Y, Qin H, Li K',
+      source: {
+        url: '',
+        value: 'Europe PMC',
+        name: ''
+      }
+    }
```

Comment:
The likely shape of a publication is:

```
      {
        "title": "Identification of the breast cancer susceptibility  gene BRCA2.",
        "authors": ["Wooster R", "Bignell G", "Lancaster J", "Swift S", "Seal S", "Mangion J", "Collins N", "Gregory S", "Gumbs C", "Micklem G", "Barfoot R", "Hamoudi R", "Patel S", "Rice C", "Biggs P", "Hashim Y", "Smith A", "Connor F", "Stratton MR"],
        "source_url": "http://europepmc.org/article/MED/8091231",
        "source": {
          "name": "Europe PMC",
          "url": "http://europepmc.org/"
        }
      }
```
Although this shape has not yet been confirmed, we can be almost sure that:
- Authors will go in the `authors` field (definitely not in "description")
- The authors field will be an array
- The elements of the authors array will either be strings `"Sun Y", "Shi T" ...` or objects `{last_name: "Sun", first_name: "T"}`. Strings are more likely
- link to the resource itself will be outside of the `source` object; not sure whether it will say `source_url` or `source_uri`

Reply:
Thanks. I will update the author. Regarding the other keys, shall we leave it as it is until it is decided?